config/HELICSConfig.cmake.in

```diff
@@ -54,6 +54,8 @@ set(CMAKE_MODULE_PATH ${PACKAGE_PREFIX_DIR}/@HELICS_CMAKECONFIG_INSTALL_DIR@)
 
 include(CMakeFindDependencyMacro)
 
+include(extraMacros)
```

Comment:
Why do we need the extraMacros?

Reply:
The SHOW_VARIABLE macro in addBoost.cmake was defined in extraMacros — got an error without it included.